client/modules/datafiles/src/projects/ProjectCitation/ProjectCitation.tsx

```diff
@@ -7,6 +7,41 @@ import {
 import { MetricsModal } from '../modals/MetricsModal';
 import styles from './ProjectCitation.module.css';
 
+const useClarivateMetrics = (doi: string, shouldFetch = true) => {
```

Comment:
This should be refactored to a react-query hook in the `hooks` module so we don't have to maintain our own loading/error states.

Reply:
created useClarivateMetrics.ts and added it index.ts